src/silo/preprocessing/preprocessor.cpp

```diff
@@ -682,6 +684,41 @@ void Preprocessor::buildMetadataStore(
    }
 }
 
+template <typename SymbolType>
+ColumnFunction Preprocessor::createRawReadLambda(
+   ZstdDecompressor& decompressor,
+   silo::SequenceStorePartition<SymbolType>& sequence_store
+) {
+   return {"read", [&sequence_store, &decompressor](size_t /*row_id*/, const duckdb::Value& value) {
+              ReadSequence& target = sequence_store.reserveRead();
```

Comment:
I think I find the name `reserveRead` a bit misleading. It makes sense once you know what it does, but it doesn't help you in understanding it.
Maybe something like `appendNewSequenceRead` would be better?